src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val verboseAuditTarget: PipelineTable = PipelineTable(
+      name = "verboseAuditTarget_gold",
+      _keys = Array("organization_id", "timestamp", "unixTimeMS"),
+      config,
+      _mode = WriteMode.merge,
+      partitionBy = Seq("organization_id"),
+      incrementalColumns = Array("unixTimeMS"),
+      zOrderBy = Array("notebookId", "unixTimeMS")
+    )
+
+    lazy private[overwatch] val verboseAuditViewTarget: PipelineView = PipelineView(
+      name = "verboseAuditTarget",
```

Comment:
change name to be appropriate for whatever you wind up naming the gold table, just remove _gold